### ISSUE TITLE
import error on deb9

### ISSUE THREAD

**Issue by ghost** [baoboa/pyqt5#45]
while on pc1 venv install / run works fine, on pc2 it breaks with:
```
(venv) user@debian:~/foo/workspace_python/combs/monitor2$ pip install -r requirements.txt
Requirement already satisfied: cycler==0.10.0 in ./venv/lib/python3.5/site-packages (from -r requirements.txt (line 1)) (0.10.0)
Requirement already satisfied: kiwisolver==1.0.1 in ./venv/lib/python3.5/site-packages (from -r requirements.txt (line 2)) (1.0.1)
Requirement already satisfied: matplotlib==2.2.2 in ./venv/lib/python3.5/site-packages (from -r requirements.txt (line 3)) (2.2.2)
Requirement already satisfied: numpy==1.15.0 in ./venv/lib/python3.5/site-packages (from -r requirements.txt (line 4)) (1.15.0)
Requirement already satisfied: PyOpenGL==3.1.0 in ./venv/lib/python3.5/site-packages (from -r requirements.txt (line 5)) (3.1.0)
Requirement already satisfied: pyparsing==2.2.0 in ./venv/lib/python3.5/site-packages (from -r requirements.txt (line 6)) (2.2.0)
Requirement already satisfied: PyQt5==5.11.2 in ./venv/lib/python3.5/site-packages (from -r requirements.txt (line 7)) (5.11.2)
Requirement already satisfied: PyQt5-sip==4.19.12 in ./venv/lib/python3.5/site-packages (from -r requirements.txt (line 8)) (4.19.12)
Requirement already satisfied: pyqtgraph==0.10.0 in ./venv/lib/python3.5/site-packages (from -r requirements.txt (line 9)) (0.10.0)
Requirement already satisfied: python-dateutil==2.7.3 in ./venv/lib/python3.5/site-packages (from -r requirements.txt (line 10)) (2.7.3)
Requirement already satisfied: pytz==2018.5 in ./venv/lib/python3.5/site-packages (from -r requirements.txt (line 11)) (2018.5)
Requirement already satisfied: redis==2.10.6 in ./venv/lib/python3.5/site-packages (from -r requirements.txt (line 12)) (2.10.6)
Requirement already satisfied: six==1.11.0 in ./venv/lib/python3.5/site-packages (from -r requirements.txt (line 13)) (1.11.0)
Requirement already satisfied: setuptools in ./venv/lib/python3.5/site-packages (from kiwisolver==1.0.1->-r requirements.txt (line 2)) (32.3.1)
(venv) user@debian:~/foo/workspace_python/combs/monitor2$ python3 src/main.py
Traceback (most recent call last):
  File "src/main.py", line 9, in <module>
    from pyqtgraph.Qt import QtGui, QtCore
  File "/home/user/foo/workspace_python/combs/monitor2/venv/lib/python3.5/site-packages/pyqtgraph/__init__.py", line 13, in <module>
    from .Qt import QtGui
  File "/home/user/foo/workspace_python/combs/monitor2/venv/lib/python3.5/site-packages/pyqtgraph/Qt.py", line 153, in <module>
    from PyQt5 import QtGui, QtCore, QtWidgets, uic
ImportError: /home/user/foo/workspace_python/combs/monitor2/venv/lib/python3.5/site-packages/PyQt5/QtGui.so: undefined symbol: PySlice_AdjustIndices
(venv) user@debian:~/foo/workspace_python/combs/monitor2$ python -V
Python 3.5.3
```

any further info need, i will add it right away

**Comment by jakabk** [comment]
Me too. Debian Stretch (9)

With python3.5.3 works the previous version only. (```pip install "PyQt5<5.11"```).
With python3.6.5 (used with [pyenv](https://github.com/pyenv/pyenv)) works without problems.

**Comment by The-Compiler** [comment]
That's already reported and fixed: https://www.riverbankcomputing.com/pipermail/pyqt/2018-July/040512.html

**Comment by baoboa** [comment]
if your are using pip you should look at pypi.org , this is not related with this repo

i will update this repo soon

**Comment by jakabk** [comment]
I don't understand what should I do.
From pypi I can download a broken wheel for python3.5.
Python 3.5 is not an old version of python as written in @The-Compiler link because debian stable use it as default.
On pypi I don't see possibilities to report broken wheel.

**Comment by The-Compiler** [comment]
3.5.3 is an old version, as the newest 3.5 version is 3.5.6 and it works fine there. Unfortunately, Debian Stable (and Ubuntu) likes keeping around outdated software :wink:

You pretty much have three options:

- Upgrade to Python 3.5.6 (or 3.6/3.7)
- Stick with PyQt 5.10.1 for now
- Wait for PyQt 5.11.3 (which will probably be released after Qt 5.11.2 is)

**Comment by jakabk** [comment]
I am aware of these solutions, see my previous post.
My question was about pypi, and the broken wheel.

Since the debian/ubuntu based distros are the 2-6 [most popular distros](https://distrowatch.com/), it is worth to supporting them.

**Comment by The-Compiler** [comment]
The PyQt author already mentioned that he's not interested in doing a new release for this alone, and it's going to be fixed with 5.11.3 anyways. The current wheel probably should stay there as it works fine everywhere else.

**Comment by jakabk** [comment]
Ok ,I understand wontfix.
